.circleci/cb-publish-step-2-verdaccio.sh

```diff
@@ -8,8 +8,15 @@ function lernaPublishExitOnFailure {
   set -e
   # run lerna publish with the args that were passed to this function
   # duplicate stdout to a temp file
-  # grep the temp file for the lerna err token and return exit 1 if found (-v option inverts grep exit code)
-  npx lerna publish "$@" | tee /tmp/publish-results && grep -qvz "lerna ERR!" < /tmp/publish-results
+  npx lerna publish "$@" | tee /tmp/publish-results
+  # grep the temp file for the lerna err token and return exit 1
+  number_of_lerna_errors=$(grep -c "lerna ERR" /tmp/publish-results)
```

Comment:
`grep -c` returns count of matches.

Reply:
I also removed `!` from regex.
It seems to be special character that leads to weird results.
![image](https://github.com/aws-amplify/amplify-cli/assets/5849952/46707f9e-1a1a-46e3-a32a-a126eb79aa35)

Reply:
![image](https://github.com/aws-amplify/amplify-cli/assets/5849952/e02a328c-8a51-49df-aba1-db1009f11f7e)